static-assets/components/cstudio-forms/forms-engine.js

```diff
@@ -1917,6 +1917,36 @@ var CStudioForms =
             YAHOO.util.Event.addListener(window, 'beforeunload', unloadFn, me);
             YAHOO.util.Event.addListener(closeButtonEl, 'click', cancelFn, me);
 
+            const canEdit = CrafterCMSNext.system.store
+              .getState()
+              .user.permissionsBySite[CrafterCMSNext.system.store.getState().sites.active].includes('content_write');
```

Comment:
Don't we have a set of constants for these strings? What happens when we change these strings in the back?

Reply:
I think we don´t have constants for these strings, another option is use the item available actions

Reply:
That's even better, let's use available actions.